package week7_abstract_interfaces.assignment_problems;

public class LetterNote extends DeliveryNote {

    private String trackingId;

    public LetterNote(String trackingId) {
        this.trackingId = trackingId;
    }

    @Override
    public String confirmDelivery() {
        return "Letter " + trackingId + " delivered";
    }
}
